js-exercises/sum-even-args/sumEvenArgs.js

```diff
@@ -0,0 +1,9 @@
+const sumEvenArgs = (...args) => {
+  return args.filter((v, i) => i % 2 === 1).reduce((a, v) => a + v);
```

Comment:
Could have named the predicate inside the filter separately.
For example,
`const isEven = (v, n) => n % 2 === 1;`